docs/windows_subsystem_for_linux.md

```diff
@@ -0,0 +1,69 @@
+# Building under the W.S.L.
+
+Users of Microsoft Windows 10 and 11 may wish to build their Bevy projects under the *Windows Subsystem for Linux*, using Linux-based tooling, for a variety of reasons and there are several approaches to doing so.
+
+The first set of approaches involve cross-compilation: building Windows artefacts from the Linux developer environment. These artefacts will be Windows executables and will run natively on the Windows host or may even be deployed to users who run Windows, given some considerations. As such, they will be able to present first-class desktop windows and have native access to the GPU, sound and other relevant hardware.
+
+The second set of approaches exploit the capability of recent WSL 2 versions to run GUI applications – available under Windows 11 *and* under Windows 10, given updates to the WSL 2. These approaches produce Linux executables that will not likely be deployable but will run, directly, from the WSL's virtualised filesystem. Some configuration is required and other considerations should be addressed.
+
+## Cross-Compilation
+
+Cross-compiling applications for Windows, under the WSL, is an appealing option and is no different from [cross-compilation for Windows](https://bevy-cheatbook.github.io/setup/cross/linux-windows.html) under *any* Linux environment – at least at build time. A brief summary of cross compilation approaches follows in sub-sections.
```

Comment:
I'd prefer if the instructions for cross compilation were also described here.

Reply:
@IceSentry — I would agree that inclusion of cross-compilation notes would be hugely valuable. At the moment, those notes do exist in a rather shallow form in the Cheat Book as far as I know but expanding them and adopting them into the core documentation would certainly get my vote. Cross compilation of Bevy applications must surely be a key requirement for many teams.

That said, however, I wouldn't really think that they belong in my new `.md` file but rather in a separate file specifically addressing the topic – one that could be linked in from my own added notes and elsewhere.

Cross-compilation would be a concern that is more generic than the exploitation of the WSL. That said, I do consider further writing on cross-compilation to be arguably out of scope for this change request and, also, as I am about to comment right here, I'm sadly out of time to add it.